server/verify/verify.go

```diff
@@ -27,8 +27,7 @@ import (
 	"go.uber.org/zap"
 )
 
-const ENV_VERIFY = "ETCD_VERIFY"
-const ENV_VERIFY_ALL_VALUE = "all"
+const ENV_VERIFY_VALUE_STORAGE_WAL verify.VerificationType = "storage_wal"
```

Comment:
Although golang prefers camel case instead of snake case, personally I think it should be OK if we keep it consistent.